query/tests/it/servers/http/http_query_handlers.rs

```diff
@@ -684,7 +684,7 @@ async fn get_uri(ep: &EndpointType, uri: &str) -> Response {
             .finish(),
     )
     .await
-    .unwrap_or_else(|err| err.as_response())
+    .unwrap_or_else(|err| err.into_response())
```

Comment:
@PsiACE sorry, did not thought you review so fast, rebased only to merge fix here

Reply:
ok, but it looks like our unit tests still need some fixes.